### ISSUE TITLE
LSS startet nicht

### ISSUE THREAD

**Issue by manu1705** [LSS-Manager/lss-manager-v3#414]
Hallo, habe das script nach Anleitung installiert. Nur startet das nicht.
Safari / Beta 3.3.1.

**Comment by KBOE2** [comment]
Was genau meinst du mit "startet das nicht"?
Wenn du angemeldet bist, müsste oben in der Navigationsleiste ein indikator zu sehen sein, gibt es den bei dir?
![grafik](https://user-images.githubusercontent.com/27238105/49463682-56cd1b80-f7f9-11e8-8322-1eb980bc2250.png)

**Comment by manu1705** [comment]
Ja, das kommt. Aber die Grafik passt nicht. Muss ich erst Plug In installieren? Wie geht das?

**Comment by KBOE2** [comment]
Wenn du auf das LSS-Manager klickst, öffnet sich ein Popup. Dann wählst du "Appstore" und aktivierst die PlugIns, die du haben/testen willst.
